testing/scenario_app/android/FontManifest.json

```diff
@@ -0,0 +1 @@
+[{"family":"Roboto","fonts":[{"asset":"Roboto-Medium.ttf"}]}]
```

Comment:
This one might cause problems since we're not bundling Roboto also - we could though since its an asset somewhere.

You'd need to add the font file to `out/fonts/Roboto-Medium.ttf"`